src/ui/setup/helpers.ts

```diff
@@ -55,6 +57,7 @@ export async function setupAppHelper(store: UIStore) {
     prefs,
     features,
     asyncStore,
+    debuggerAsyncStore,
```

Comment:
This seems unrelated to this PR. What is it for?

Reply:
Was this change intentional to land? @jcmorrow

Reply:
Oh, I totally forgot that this was even in here. It seems unrelated. I'll PR it back out.